server/src/data/index.js

```diff
@@ -0,0 +1,9 @@
+const { MeasurablesData } = require('./measurables-data');
```

Comment:
I'm not familiar with the naming convention of this being called ``data``. Where does that come from?

Reply:
I don't have proof links but in couple projects I'm working on we have a middleware, data and model layers. Middleware handles requests and uses data layer to do CRUD operations. Data layer is an abstract layer, it interacts with models. Data isn't aware of requests, responses and database stuff.

Reply:
Interesting. I've heard similar things be called Services or similar. Data is fine for now; the organization is more important than the naming.